frontend/core-ui/src/modules/automations/components/list/AutomationColumns.tsx

```
@@ -44,6 +44,7 @@ export const getAutomationColumns: (
 ) => ColumnDef<TAutomationRecordTableColumnDefData>[] = (t) => [
   {
     id: 'more',
+    size: 33,
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)